cw/tvla.py

```diff
@@ -787,18 +668,18 @@ def main():
                 histograms = np.concatenate((histograms[:]), axis=3)
 
             # Add up new data to potential, previously generated histograms.
-            if args.input_file is not None or i_step > 0:
+            if cfg["input_file"] is not None or i_step > 0:
                 histograms = histograms + histograms_in
 
             # Move current histograms to temp variable for next step.
             if num_steps > 1 and i_step < num_steps - 1:
                 histograms_in = histograms
 
             # Histograms can be saved for later use if output file name is passed.
-            if args.output_file is not None:
+            if cfg["output_file"] is not None:
                 log.info("Saving Histograms")
-                np.savez(args.output_file, histograms=histograms, rnd_list=rnd_list,
-                         byte_list=byte_list, single_trace = traces[1])
+                np.savez(cfg["output_file"], histograms=histograms, rnd_list=rnd_list,
+                         byte_list=byte_list)
```

Comment:
It looks like `single_trace = traces [1]` was removed from this line by accident. Could you return it?

Reply:
I returned it. Thanks